cmd/virt-chroot/tap-device-maker.go

```diff
@@ -35,7 +35,7 @@ func NewCreateTapCommand() *cobra.Command {
 			tapName := cmd.Flag("tap-name").Value.String()
 			uidStr := cmd.Flag("uid").Value.String()
 			gidStr := cmd.Flag("gid").Value.String()
-			isMultiqueueStr := cmd.Flag("multiqueue").Value.String()
+			queueNumberStr := cmd.Flag("queue-number").Value.String()
```

Comment:
I think that you can use `cmd.Flags().GetUint32("queue-number")` directly.

Reply:
Done, thanks for pointing it out.